JiraPS/Public/Remove-JiraIssue.ps1

```diff
@@ -0,0 +1,108 @@
+function Remove-JiraIssue {
+    [CmdletBinding(
+        ConfirmImpact = 'High',
+        SupportsShouldProcess
+    )]
+    param (
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline,
+            Position = 0
+        )]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript(
+            {
+                if (("JiraPS.Issue" -notin $_.PSObject.TypeNames) -and (($_ -isnot [String]))) {
+                    $errorItem = [System.Management.Automation.ErrorRecord]::new(
```

Comment:
This is unfortunatly not compatible with PSv3.
Please update it, as I did with the rest of the module:

https://github.com/AtlassianPS/JiraPS/pull/273/files#diff-762c92178567422918bfa5448fbb1e52R28

Reply:
Commit 1627090ffb72762e4cc905d6b8df7bac8a60ad9c should have fixed this.